app/models/fe/date_field.rb

```diff
@@ -29,7 +29,7 @@ def response(answer_sheet = nil)
       r
     end
 
-    def display_response(answer_sheet = nil)
+    def display_response(answer_sheet = nil, humanize = false)
```

Comment:
Oh? Why is that?
On 6 Nov 2015 19:04, "Josh Starcher" <notifications@github.com> wrote:

> In app/models/fe/date_field.rb
> <https://github.com/CruGlobal/fe/pull/60#discussion_r44201256>:
>
> > @@ -29,7 +29,7 @@ def response(answer_sheet = nil)
> >        r
> >      end
> >
> > -    def display_response(answer_sheet = nil)
> > +    def display_response(answer_sheet = nil, humanize = false)
>
> I think rubocop would complain about this if it were running. I think
> unused parameters should start with _
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CruGlobal/fe/pull/60/files#r44201256>.
>